lua/warden/warden.lua

```diff
@@ -1,14 +1,11 @@
-function Warden.SetupPlayer(ply)
-	Warden.Permissions[ply:SteamID()] = {}
-	for _, id in pairs(Warden.PermissionIDs) do
-		Warden.Permissions[ply:SteamID()][id] = { global = false }
+function Warden.SetupPlayer(plyOrID)
+	if not isstring(plyOrID) then
+		plyOrID = plyOrID:SteamID()
```

Comment:
why did you merge these

Reply:
It seemed unusual for there to be two functions for this, especially when the former didn't do anything with the extra information. This function ideally only executes once per player as well.